x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rule_status.tsx

```diff
@@ -113,9 +113,11 @@ export const useRulesStatuses = (rules: Rules): ReturnRulesStatuses => {
         setLoading(false);
       }
     };
-    if (rules != null && rules.length > 0) {
+
+    if (rules.length > 0) {
```

Comment:
A bit unrelated but I notice that sometimes we use `Rule[]` and sometimes we use `Rules`. I wonder if sometimes we don't check the type to realize that `Rules` is just `Rule[]` and non nullable.

Reply:
Maybe? I just know that we should not check for null if it can never be null or update the types to reflect the runtime reality. In this case it looks like it can never be null.